style-dictionary/visual-refresh/color-severity.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { expandColorDictionary } from '../utils';
+import { StyleDictionary } from '../utils/interfaces';
+
+const tokens: StyleDictionary.ColorSeverityDictionary = {
+  colorSeverityDarkRed: { light: '#870303', dark: '#d63f38' },
+  colorSeverityRed: { light: '#ce3311', dark: '#fe6e73' },
+  colorSeverityOrange: { light: '#f89256', dark: '#f89256' },
+  colorSeverityYellow: { light: '#f2cd54', dark: '#f2cd54' },
+  colorSeverityGrey: '{colorGrey550}',
```

Comment:
nit: Since we are only using each of these in one place (and they are in this file), and have no intention of exposing these tokens, I think we could consider simplifying by removing this layer. The risk is that we accidentally use one of these internally when we should've used the semantic token and it makes finding dependencies more difficult. 
Not a huge risk. If you think it is helpful to have the color in the name, you can leave it as is. And just add a comment and/or include it in the token metadata specifying that these should not be used directly.

Reply:
I removed these from the metadata, so I'll add a comment here. I think it could be helpful in the future in case we add other reds / yellows / etc. to the palette to have the two layers.